core/modules/menu/menu.admin.inc

```diff
@@ -14,7 +14,7 @@ function menu_overview_page() {
   $rows = array();
   foreach ($menus as $menu) {
     $row = array();
-    $row[] = theme('menu_admin_overview', array('title' => $menu['title'], 'name' => $menu['menu_name'], 'description' => $menu['description']));
+    $row[] = theme('admin_info', array('label' => $menu['title'], 'machine' => $menu['menu_name'], 'description' => $menu['description']));
```

Comment:
If we're no longer calling `theme_menu_admin_overview` at all, maybe it should be removed entirely? Seems unlikely that another place would actually call that theme function.

Reply:
removed.